nlmod/read/bgt.py

```diff
@@ -321,3 +326,517 @@ def get_bgt_layers():
     resp = requests.get(url, timeout=1200)  # 20 minutes
     data = resp.json()
     return [x["featuretype"] for x in data["timeliness"]]
+
+
+def get_bronhouder_names() -> Dict[str, str]:
+    """Get the names of the bronhouders of the BGT data.
+
+    Returns
+    -------
+    dict
+        A dictionary with the bronhouder codes as keys and the names as values.
+
+    Notes
+    -----
+        The bronhouder names are retrieved from
+        https://www.kadaster.nl/-/bgt-bronhoudercodes. The `Toelichting` sheet
+        in the .ods file gives the changes compared to the old file. If changes
+        are made, please add them manually to the bgt_bronhouder_names
+        dictionary. A test is added for to check if the dictionary is up to
+        date with the latest file from the Kadaster up to date with the .ods
+        file from 2024-01-01.
+    """
+    bgt_bronhouder_names = {
```

Comment:
I would prefer to put 300 lines of data in a separate data file not in a function

Reply:
fixed